src/model/json-schema.json

```diff
@@ -4141,4 +4129,4 @@
         }
     },
     "$schema": "http://json-schema.org/draft-07/schema#"
-}
```

Comment:
Do we want to keep this as-is?

Reply:
I think we have to remove it from the schema otherwise DCR still expects a "hbImpl" property in the config

Reply:
I was only referring to the return at the end of the file!

Reply:
Oh I see!